code/modules/hydroponics/gene_modder.dm

```diff
@@ -36,15 +37,33 @@
 							/obj/item/weapon/stock_parts/console_screen = 1,
 							/obj/item/weapon/stock_parts/scanning_module = 1)
 
-/obj/machinery/plantgenes/RefreshParts()
-	rating = 0
-	for(var/I in component_parts)
-		if(istype(I, /obj/item/weapon/stock_parts))
-			var/obj/item/weapon/stock_parts/S = I
-			rating += S.rating-1
-		else if(istype(I, /obj/item/weapon/circuitboard/machine/plantgenes/vault))
-			rating += 5 // Having original alien board is +25%
-	max_extract_pot = initial(max_extract_pot) + rating*5
+/obj/machinery/plantgenes/RefreshParts() // Comments represent the max you can set per tier, respectively. seeds.dm [219] clamps these for us but we don't want to mislead the viewer.
+	for(var/obj/item/weapon/stock_parts/manipulator/M in component_parts)
+		max_potency = initial(max_potency) + (M.rating**3) // 53,59,77,100 	 Clamps at 100
```

Comment:
I need to clamp this as it will most certainly go over 100

Silly @XDTM I already did what you asked, this is why I should not use mobile or code in the early morning!